com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl

```diff
@@ -14,12 +14,12 @@
             
         #define APPLY_NORMALS_LIGHTING(input, lightColor)\
             half4 normal = SAMPLE_TEXTURE2D(_NormalMap, sampler_NormalMap, input.screenUV);\
-            float3 normalUnpacked = UnpackNormal(normal);\
+            half3 normalUnpacked = UnpackNormal(normal);\
             lightColor = lightColor * saturate(dot(input.lightDirection.xyz, normalUnpacked));
     #else
         #define NORMALS_LIGHTING_COORDS(TEXCOORDA, TEXCOORDB) \
-            float4	positionWS : TEXCOORDA;\
-            float2	screenUV   : TEXCOORDB;
+            half4	positionWS : TEXCOORDA;\
+            half2	screenUV   : TEXCOORDB;
```

Comment:
if this is not normalized uv you need to double check you have precision to store it in half.